pages/1.11/installing-upgrading/custom/cli-installer/oss/index.md

```diff
@@ -0,0 +1,408 @@
+---
```

Comment:
This file and the enterprise version have very similar steps, and although the docs look different I think the process is the same. Please review and merge if possible. If not please combine the pages and list the processes sequentially.